### PR TITLE
Implemented stats helper

### PR DESCRIPTION
**Pull request by runz0rd**
For #487

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=506) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-php?pullRequest=506) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-php?pullRequest=506) it.</sub>

File: lib/helpers/stats/Stats.php [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+    /**
+     * @param array $array
+     * @return bool
+     */
+    protected function isNumeric(array $array) {
+        return array_keys($array) == range(0, count($array) - 1);
+    }
+
+}
```
Comment (shonjord) [inline lib/helpers/stats/Stats.php]: Missing newline at the EOF

**Comment by shonjord** [comment]
Excellent PR, but can you make it compatible with PSR2?

**Comment by runz0rd** [comment]
Sure, coming up.

**Comment by runz0rd** [comment]
Note: The ```\SendGrid\Client``` will have issues with forming the query strings with requests that contain arrays such as ```Stats::getSubuser()``` with subusers being a numericArray inside it.
The ```\SendGrid\Client::buildUrl()``` uses http_build_query(), and will output somwthing like subusers[0]=x&subusers[1]=x...
One solution for this would be to replace the string in question after http_build_query():
```
$path .= '?' . http_build_query($queryParams);
$path = preg_replace('/%5B[0-9]+%5D/simU', '', $path);
```

**Comment by thinkingserious** [comment]
Hello @runz0rd,<br /><br/>Thanks again for the PR!<br />                                              <br />We appreciate your contribution and look forward to                                              continued collaboration. Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**